crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -1419,6 +1419,22 @@ impl EmptyChunk {
     }
 }
 
+/// The raw representation of a linked chunk, as persisted in storage.
+#[derive(Debug)]
+pub struct RawLinkedChunk<Item, Gap> {
```

Comment:
Nice to get it here.

Question: `Chunk` contains `previous`, `next`, `identiifer` and `content`. So… technically, this type should be `RawChunk`, not `RawLinkedChunk`. Sorry for the rebase it will incur but I think `RawChunk` is the correct naming.

Reply:
No worries, it's a good point, and I can rename in a separate commit. Thanks.